### PR TITLE
Fix #45 Remove getowner-polyfill dependency

### LINKED ISSUE #45: Fix getowner deprecation
With a new release of ember (I'm thinking 2.10) this deprecation warning is happening everywhere

```
ember-getowner-polyfill is now a true polyfill. Use Ember.getOwner directly instead of importing from ember-getowner-polyfill
```

Pull request incoming!

### PR DESCRIPTION
**Pull request by Frozenfire92**
issue #45

### CODE REVIEW

**Comment by Frozenfire92** [comment]
I'm unsure on the best way to make your test suite pass for older versions of ember. Is there a way that you can conditionally include a dependency based on the version of ember used?

File: package.json [hunk 1]
```diff
@@ -47,7 +47,6 @@
     "route action"
   ],
   "dependencies": {
-    "ember-getowner-polyfill": "^1.0.0",
```
Comment (rwjblue) [inline package.json]: The dep is still required (for backwards compat to < ember@2.3), can you bring it back and bump to the latest version?

**Comment by rwjblue** [comment]
The deprecation isn't about having the polyfill addon as a dependency, it's deprecating importing it manually as a function (instead of using Ember.getOwner directly).

File: package.json (hunk 1 above)
Comment (Frozenfire92) [inline package.json]: Gotcha, should be fixed now!

**Comment by poteto** [comment]
@rwjblue Happy with the update?

**Comment by rwjblue** [comment]
Thank you!

**Pull request merged**